benchmark/benchmarks.jl

```diff
@@ -1,25 +1,66 @@
 using BenchmarkTools, ONSAS, Suppressor
 
+include("aux_bench.jl")
```

Comment:
maybe renamre `aux_bench.jl` to `utils.jl` or `io.jl` ?

Reply:
Done